frontend/src/scenes/authentication/SupportModalButton.tsx

```diff
@@ -1,20 +1,23 @@
 import { LemonButton } from '@posthog/lemon-ui'
 import { useActions, useValues } from 'kea'
 import { supportLogic } from 'lib/components/Support/supportLogic'
-import { SupportModal } from 'lib/components/Support/SupportModal'
 import { IconBugShield } from 'lib/lemon-ui/icons'
 import { preflightLogic } from 'scenes/PreflightCheck/preflightLogic'
 
 export function SupportModalButton({ name, email }: { name?: string; email?: string }): JSX.Element | null {
-    const { openSupportLoggedOutForm } = useActions(supportLogic)
+    const { openSupportForm } = useActions(supportLogic)
     const { preflight } = useValues(preflightLogic)
 
     return preflight?.cloud ? ( // We don't provide support for self-hosted instances
         <>
             <div className="text-center">
                 <LemonButton
                     onClick={() => {
-                        openSupportLoggedOutForm(name, email, null, 'login')
+                        openSupportForm({
+                            name,
+                            email,
+                            target_area: 'login',
+                        })
```

Comment:
Given the button says "Report an issue", it might be a nice default

```suggestion
                        openSupportForm({
                            name,
                            email,
                            kind: 'bug',
                            target_area: 'login',
                        })
```